js-miniapp-sdk/src/modules/secure-storage.ts

```diff
@@ -0,0 +1,50 @@
+import { MiniAppError } from '../../../js-miniapp-bridge/src';
+import { getBridge } from '../utils';
+import {
+  MiniAppSecureStorageKeyValues,
+  SecureStorageResponseStatus,
+  MiniAppSecureStorageSize,
+} from '../../../js-miniapp-bridge/src/types/secure-storage';
+
+interface SecureStorageProvider {
```

Comment:
We also need to define event types in `MiniAppEvents` for something like `miniappsecurestorageready` and `miniappsecurestorageloaderror`. These will be used for letting the Mini App know when the Secure Storage is ready to use (or cannot be used because it failed to load).

And then I think we could add a convenience function that does something like this (I had originally defined only `onReady` for handling both ready and error, but maybe it makes sense to have both `onReady` and `onLoadError`):

```
MinApp.secureStorage.onReady(onReady: () => undefined) {
  window.addEventListener('miniappsecurestorageready', () => onReady())
}

MinApp.secureStorage.onLoadError(onLoadError: (error: MiniAppError) => undefined) {
  window.addEventListener('miniappsecurestorageloaderror', (error) => onLoadError(error))
}
```